docs/deployment/helm.md

```diff
@@ -149,6 +149,8 @@ Chart parameters are available.
 
 | Name                                                | Type   | Default                                             | Description                                                                                                                                                                                                  |
 | --------------------------------------------------- | ------ | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `crds.install`                                      | bool   | false                                               | Toggle to install and upgrade CRDs from a subchart. Make sure to use it with `--skip-crds` to avoid conflicts. [More info about limitations on CRDs in Helm 3](https://helm.sh/docs/topics/charts/#limitations-on-crds) |
```

Comment:
If i'm not mistaken, I think the `--skip-crds` flag is used for the `crds` folder and not the separate helm chart. so I don't think that they're related. Please correct if I'm wrong.

Reply:
You're right about the flag 👍 

The problem happens only when the chart is installed by the first time. In that scenario: `helm install xxxx --skip-crds --set crds.install=true`

(*apologies for the oversimplification above*)

Otherwise, Helm will install CRDs from both places. Again, this only happens the first time. Helm won't care about the CRDs folder during upgrades (nor uninstall)

Reply:
Are you keeping an additional copy of the CRD in the /crds directory only for the initial installation of the chart?
I'm wondering if it might be too strict to always require using `--skip-crds --set crds.install=true`.

Reply:
My take was make the new approach optional. That is, if you do nothing, then the CRDs get installed as they are now.
By the other hand, if you want to taste the new approach, then `--skip-crds --set crds.install=true`.

I wonder if this is a good compromise, or if we should go the sub-chart with CRDs approach in one single shot. WDYT @fmuyassarov 

Regarding additional copies, all of them are automatically generated.